dji_robomaster_ep_driver/src/dji_robomaster_ep_driver_node_main.ros2.cpp

```diff
@@ -0,0 +1,16 @@
+#include <memory>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include "dji_robomaster_ep_driver_node.ros2.hpp"
+
+int main(int argc, char * argv[])
```

Comment:
This should be `char** argv` to be consistent with existing code

Reply:
Done in https://github.com/ToyotaResearchInstitute/tri_hardware_drivers/pull/9/commits/d690864fb8503b1379022f5d42b6c5ac5618bb4e.